libbindgen/src/ir/context.rs

```diff
@@ -1066,3 +1102,38 @@ impl<'ctx, 'gen> Iterator for WhitelistedItemsIter<'ctx, 'gen>
         Some(id)
     }
 }
+
+pub struct RelatedItemsIter<'ctx, 'gen>
```

Comment:
Since this iterator is going to be specific to finding BFS paths of dangling items, lets call it `AssertNoDanglingItemIter`.